data/chapters.js

```diff
@@ -0,0 +1,214 @@
+const chapters = [
```

Comment:
What's this being used for ?

Reply:
In the `addChapters` utility function to seed the  database.